api/v1alpha08/sonataflow_types.go

```diff
@@ -115,6 +644,8 @@ type SonataFlowSpec struct {
 	// Resources workflow resources that are linked to this workflow definition.
 	// For example, a collection of OpenAPI specification files.
 	Resources WorkflowResources `json:"resources,omitempty"`
+	// PodSpecTemplate describes the deployment details of this SonataFlow instance.
+	PodSpecTemplate FlowPodSpecTemplate `json:"podSpecTemplate,omitempty"`
```

Comment:
Following the analogy with other workload resources like Job, Deployment, etc, maybe we can name this filed just Template instead of PodSpecTemplate